app/Providers/RouteServiceProvider.php

```diff
@@ -29,13 +29,9 @@ public function boot()
      */
     public function map()
     {
-        Route::middleware(['api'])->prefix('/api/user')
-              ->namespace($this->namespace . '\API\User')
-              ->group(base_path('routes/api.php'));
-
-        Route::middleware(['api'])->prefix('/api/admin')
-              ->namespace($this->namespace . '\API\Admin')
-              ->group(base_path('routes/api-admin.php'));
+//        Route::middleware(['api'])->prefix('/api/user')
```

Comment:
Remove instead of commenting out

Reply:
Theres a second PR coming in behind this to implement all of those routes.